source/brailleDisplayDrivers/handyTech.py

```diff
@@ -581,10 +602,47 @@ def __init__(self, port="auto"):
 			self._dev.close()
 
 		else:
+			wx.CallAfter(self.destroy_message_window)
 			raise RuntimeError("No Handy Tech display found")
 
+	def create_message_window(self):
+		try:
+			self._messageWindow = InvisibleDriverWindow(self)
+		except:
+			log.debugWarning("", exc_info=True)
+	def destroy_message_window(self):
+		try:
+			self._messageWindow.destroy()
+		except:
+			log.debugWarning("", exc_info=True)

```

Comment:
Make sure that if you move window message registering to the driver, that it is properly unregistered.